nodes/textures/triplanar.py

```diff
@@ -0,0 +1,24 @@
+import bpy
+from ..base import LuxCoreNodeTexture
+
+
+class LuxCoreNodeTriplanar(bpy.types.Node, LuxCoreNodeTexture):
+    bl_label = "Triplanar mapping"
+    bl_width_default = 160
+
+    def init(self, context):
+        self.add_input("LuxCoreSocketColor", "Texture 1", [0.8, 0.0, 0.0])
+        self.add_input("LuxCoreSocketColor", "Texture 2", [0.0, 0.8, 0.0])
+        self.add_input("LuxCoreSocketColor", "Texture 3", [0.0, 0.0, 0.8])
```

Comment:
Since these textures correspond to coordinate axis X, Y, Z, maybe we should name them as such so the user knows which texture is on which axis?
By the way, are they aligned to the local (object) or world coordinate system?

Reply:
I suggested @Alpistinho, in LuxCore pull request, to use a TextureMapping3D (GlobalMapping3D or LocalMapping3D) to map the hit point so the user can manipulate the trilinear texture as he wants.

Reply:
Ok, so the node is missing a "LuxCoreSocketMapping3D" input.

You can look at another node how to add this, for example here:
https://github.com/LuxCoreRender/BlendLuxCore/blob/7f15ec345c1df215e1389b05bda9ae28663bb608/nodes/textures/blenderclouds.py#L26
And here it's exported:
https://github.com/LuxCoreRender/BlendLuxCore/blob/7f15ec345c1df215e1389b05bda9ae28663bb608/nodes/textures/blenderclouds.py#L42
https://github.com/LuxCoreRender/BlendLuxCore/blob/7f15ec345c1df215e1389b05bda9ae28663bb608/nodes/textures/blenderclouds.py#L53